src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -1024,22 +995,22 @@ def merge_entry_points(
             for ep_name, entry_point in entry_points.items():
                 merged[category][ep_name].append((description_of_source, entry_point))
 
-    def _merge_entry_point(
+    def _check_entry_point_single_source(
         category: str, name: str, entry_points_with_source: List[Tuple[str, str]]
-    ):
+    ) -> Tuple[str, str]:
         if len(entry_points_with_source) > 1:
             raise ValueError(
                 f"Multiple entry_points registered for {category} {name} in: "
                 f"{', '.join(ep_source for ep_source, _ in entry_points_with_source)}"
             )
-        for _, entry_point in entry_points_with_source:
-            return f"{name}={entry_point}"
+        _, entry_point = entry_points_with_source[0]
+        return name, entry_point
```

Comment:
Excellent change! This is much more obvious what it does. Nice :D

Reply:
This is the most rewarding part to me, tbh. Learning to write comprehensible code. Normally, it's just I that have to grok my code, and my peers are lost in the mist of what it is supposed to do in the first place, so there is no real pressure to be super clear with the intentions. But, it will be a lot easier to maintain in the long run, so well worth spending a few minutes to get there. So, that's what I'm happy about, to be able to get there with more ease than before.. ;)